### PR TITLE
added migration for publishing

### PR DESCRIPTION
**Pull request by danrot**
| Q | A
| --- | ---
| Bug fix? | no
| New feature? | yes
| BC breaks? | no
| Deprecations? | no
| Fixed tickets | none
| Related issues/PRs | none
| License | MIT
| Documentation PR | none

#### What's in this PR?

This PR contains a migration which creates and correctly fills the workspace for published content.

#### Why?

Because copying the entire tree over to the other workspace also contains test pages.

### CODE REVIEW

File: src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php [hunk 1]
```diff
@@ -0,0 +1,94 @@
...
+    public function up(SessionInterface $session)
+    {
+        $defaultSession = $this->container->get('sulu_document_manager.default_session');
+        $liveSession = $this->container->get('sulu_document_manager.live_session');
+        $session->getWorkspace()->createWorkspace($liveSession->getWorkspace()->getName());
+        $queryManager = $liveSession->getWorkspace()->getQueryManager();
+
+        $fileName = tempnam(sys_get_temp_dir(), 'sulu-publishing');
+        $file = fopen($fileName, 'w');
+        $defaultSession->exportSystemView('/cmf', $file, false, false);
```
Comment (danrot) [inline src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php]: @chirimoya Do we agree that copying the `cmf` node is enough? Then we could say we only have the `jcr:versions` node on the draft workspace, and future migrations have to adapt both workspaces.

Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php]: ack

File: src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php [hunk 2]
```diff
@@ -0,0 +1,98 @@
...
+            $i = 0;
+            foreach ($rows->getNodes() as $node) {
+                $propertyPrefix = 'i18n:' . $localization . '-*';
+                // remove all localized system properties from the node
+                foreach ($node->getProperties($propertyPrefix) as $property) {
+                    $property->remove();
+                }
+
+                // remove all localized content properties from the node
+                foreach ($node->getProperties($propertyPrefix) as $property) {
```
Comment (chirimoya) [inline src/Sulu/Bundle/ContentBundle/Resources/phpcr-migrations/Version201607181533.php]: remove

**Comment by chirimoya** [comment]
ACK

**Pull request merged**